packages/d3fc-webgl/src/shaders/vertexShaderSnippets.js

```diff
@@ -247,12 +247,45 @@ export const area = {
 };
 
 export const boxPlot = {
-    header: `attribute float aXValue;
-        attribute float aYValue;
-        attribute float aXDirection;
-        attribute float aYDirection;
-        attribute float aBandwidth;
+    header: `
+        attribute float aXValue;
+        attribute float aHigh;
+        attribute float aUpperQuartile;
+        attribute float aMedian;
+        attribute float aLowerQuartile;
+        attribute float aLow;
+        attribute float aBandwidthNew;
```

Comment:
Strange name?

Reply:
Indeed... A relic of the refactor I think.